andela/wg-baqi
# #166151263 Upgrade Nutrition Diary

### What does this PR do?
Enables the user to enter the food he/she ate during the day, in addition to what was planned with the nutritional plan.

### Description of Task to be completed?
Initially, a user could enter into the nutrition plan the food they plan to eat. The following features have been added to the application:
  - [x] Adding an extra field for meals already eaten
  - [x] Fixing failing tests
  - [x] Adjusting the meal-plan table on the front end to display meal type column.

### How should this be manually tested?
Clone the repository to your local branch
```
$ git clone https://github.com/andela/wg-baqi.git
```
$ cd to the `wg-baqi` directory, create a virtual environment and install the dependencies
```
$ cd wg-baqi
$ virtualenv venv
$ source venv/bin/activate
$ pip install -r requirements.txt
```
Activate bootstrap, make migrations and run the server
```
$ bower install
$ invoke bootstrap-wger --settings-path ./wger/settings.py --no-start-server
$ python manage.py runserver
```
- Navigate to `Nutrition Plans` page

<img width="1047" alt="Screenshot 2019-06-20 at 08 47 13" src="https://user-images.githubusercontent.com/31995387/59824107-dc995a00-9338-11e9-824c-5467f23d5019.png">

- Click on `Add a new meal plan`. You can clearly see the option to add a Meal type as either `planned` or `consumed`

<img width="927" alt="Screenshot 2019-06-20 at 08 48 33" src="https://user-images.githubusercontent.com/31995387/59824095-d60ae280-9338-11e9-82ec-fc239baeac01.png">

- On adding the meal, the column for Meal Type can be clearly seen

<img width="816" alt="Screenshot 2019-06-20 at 08 49 58" src="https://user-images.githubusercontent.com/31995387/59824067-c7243000-9338-11e9-9a44-c19cd11b42ed.png">
